### PR TITLE
feat: npm link to npm version

### PR DESCRIPTION
**Pull request by ovflowd**
This PR simply updates the NPM piece of the Downloads Page to actually link to the NPM version that is bundled.

### Screenshot

image

### CODE REVIEW

**Comment by ovflowd** [comment]
cc @nodejs/nodejs-website

File: components/Downloads/Release/NpmLink.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import { useContext } from 'react';
+import type { FC } from 'react';
+
+import { ReleaseContext } from '@/providers/releaseProvider';
+
+import LinkWithArrow from './LinkWithArrow';
```
Comment (canerakdas) [inline components/Downloads/Release/NpmLink.tsx]:
```suggestion
import LinkWithArrow from '@/components/Downloads/Release/LinkWithArrow';
```

It looks a little strange, maybe we should update the location of the component at some point 🤔

Comment (ovflowd) [reply components/Downloads/Release/NpmLink.tsx]: True, this should probably be a Common component

**Review by canerakdas — APPROVED** [review]
LGTM

**Review by AugustinMauroy — APPROVED** [review]
LGTM !

**Review by aymen94 — APPROVED** [review]

File: next.mdx.use.mjs [hunk 2]
```diff
@@ -66,8 +66,8 @@ export const mdxComponents = {
     BitnessDropdown: BitnessDropdown,
     // Renders a drop-down menu from which the operating system can select
     OperatingSystemDropdown: OperatingSystemDropdown,
-    // Renders a npm version of the selected release
-    NpmVersion: NpmVersion,
+    // Renders a link to the NPM version of the selected release
```
Comment (bmuenzenmeyer) [inline next.mdx.use.mjs]: `npm` should never be capitalized https://github.com/npm/cli?tab=readme-ov-file#faq-on-branding

```suggestion
    // Renders a link to the npm version of the selected release
```

**Review by bmuenzenmeyer — APPROVED** [review]
tiny nit resolved.
also handled the corepack merge conflict

**Pull request merged**